FFXCS.asl

```diff
@@ -623,5 +625,35 @@ update
         //game.WriteValue(modules.First().BaseAddress+0xF3080C, 1); //Force load
     } 
 
+    if (current.roomNumber == 49 && current.storyline == 48)
+    {
+        print("Baaj Temple - Escape from Geogaesno");
+        game.WriteValue(modules.First().BaseAddress+0xD2CA90, 50); //Area ID
+        game.WriteValue(modules.First().BaseAddress+0xD2D67C, 50); //CutsceneID
+        game.WriteValue(modules.First().BaseAddress+0xF3080C, 1); //Force load
+    }
+
+    if (current.roomNumber == 63 && current.storyline == 54)
+    {
+        print("Baaj Temple - Tidus makes fire");
+        game.WriteValue(modules.First().BaseAddress+0xD2D67C, 55); //CutsceneID
+        game.WriteValue(modules.First().BaseAddress+0xF3080C, 1); //Force load
+    }
+
+    if (current.roomNumber == 63 && current.storyline == 58)
+    {
+        print("Baaj Temple - Rikku punches Tidus");
+        game.WriteValue(modules.First().BaseAddress+0xD2CA90, 71); //Area ID
+        game.WriteValue(modules.First().BaseAddress+0xD2D67C, 66); //CutsceneID
+        game.WriteValue(modules.First().BaseAddress+0xF3080C, 1); //Force load
+    }
+
+    if (current.roomNumber == 53 && current.storyline == 294)
+    {
+        print("Kilika - Sending");
+        game.WriteValue(modules.First().BaseAddress+0xD2CA90, 152); //Area ID
+        game.WriteValue(modules.First().BaseAddress+0xD2D67C, 302); //CutsceneID
```

Comment:
Skips when you screen transition to sending area, so those few steps you walk are skipped also. Didn't think it was a big deal